perma_web/perma/templates/docs/perma-link-creation.html

```diff
@@ -89,7 +89,7 @@ <h3 class="body-bh" id="citing-perma-links">Citing Perma Links</h3>
 
       <figure>
         <figcaption>An example from the <a href="http://cdn.harvardlawreview.org/wp-content/uploads/2014/06/vol127_west.pdf">Harvard Law Review</a></figcaption>
-        <img src="{{ STATIC_URL }}img/docs/citation_example.png" class="img-responsive screen-shot" alt="An example from the Harvard Law Review" />
+        <img src="{{ STATIC_URL }}img/docs/citation_example.png" class="img-responsive screen-shot" alt="Footnote 4. See, e.g., Charles P. Pierce, This Cannot Be the Way Occupy Ends, ESQUIRE: POL. BLOG (Nov 17, 2011) http://www.esquire.com/blogs/politics/occupy-wall-steret-violence-6575448, archived at https://perma.cc/48VC-ZS62 (bemoaning protestors' 'reciprocal violence' and arguing that the protest 'can't end in images of bleeding cops and tossed barricades'" />
```

Comment:
Change "steret" to "street" in the URL here and on line 106 of perma_web/perma/templates/docs/index.html.